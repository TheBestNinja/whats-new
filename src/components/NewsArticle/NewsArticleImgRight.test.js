import React from 'react';
import { shallow } from 'enzyme';
import NewsArticleImgRight from './NewsArticleImgRight'

describe('NewsArticleImgRight', () => {
  it('should match the snapshot with all data passed in correctly', () => {

    const article = {
      id: 5,
      img: 'www.101.com',
      headline: 'Hundred and One Dalmatians',
      description: 'Cruella visits their home and expresses contempt upon meeting Roger. Her initial disgust at them having a baby turns to excitement when she finds out Perdy is expecting too.',
      url: 'www.101.com'
    }

    const wrapper = shallow(
      <NewsArticleImgRight article={article} />
    )

    expect(wrapper).toMatchSnapshot()

  })
})