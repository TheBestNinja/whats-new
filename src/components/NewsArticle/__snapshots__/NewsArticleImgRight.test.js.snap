// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewsArticleImgRight should match the snapshot with all data passed in correctly 1`] = `
<article
  className="newsArticle"
  id={5}
>
  <div
    className="newsArticleContents"
  >
    <h2
      className="articleTitle"
    >
      Hundred and One Dalmatians
    </h2>
    <p
      className="articleDescription"
    >
      Cruella visits their home and expresses contempt upon meeting Roger. Her initial disgust at them having a baby turns to excitement when she finds out Perdy is expecting too.
    </p>
    <a
      className="articleLink"
      href="www.101.com"
    >
      Full Story ▶︎ 
    </a>
  </div>
  <div
    className="imageDiv"
  >
    <img
      alt="Hundred and One Dalmatians"
      className="img"
      src="www.101.com"
    />
  </div>
</article>
`;
